.github/ISSUE_TEMPLATE/update-existing-site.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Update existing site
+about: Request that an existing listed site be updated with new information.
+title: Update [site name]
+labels: update site
+assignees: ''
+
+---
+
+<!-- Before submitting this issue, please update the title to include the name of the site to be updated. 
+Submit a single issue for each site to be updated.
+Check the box below before submitting your issue to verify that you have already checked for duplicate open issues and pull requests relating to your request. -->
+
+- [ ] I have searched [open issues and pull requests](https://github.com/2factorauth/twofactorauth/issues?q=is%3Aopen). The issue I'm creating is not a duplicate of an existing open issue or pull request.
+
+### Information about the site entry to be updated: ###
+<!-- Name of the site, as currently listed on twofactorauth.org -->
+* Site name - ``
+
+<!-- Link to the main page of the site, as currently listed on twofactorauth.org -->
+* Site URL - ``
+
+<!-- Category of the site, as currently listed on twofactorauth.org -->
+* Site category - ``
+
+### Information about why the site entry should be updated: ###
+This site entry should be updated because:
+- [ ] It now supports 2FA.
+- [ ] The methods of 2FA that it supports have changed.
+- [ ] It has begun the process of implementing 2FA.
+- [ ] It no longer supports 2FA.
+- [ ] Its name has changed.
+- [ ] Its documentation link has changed.
+- [ ] Its Twitter username, Facebook page, and/or email address have changed.
+- [ ] Its logo has changed.
+- [ ] It has been acquired or absorbed by another site, and the new site is eligible to be listed on twofactorauth.org.
+- [ ] Other (please describe below).
```

Comment:
Maybe have `It fits better into a different category.` (or something like that) as well? Or is it too rare an occasion to include it here?

Reply:
Didn't even think of that! I've added it. 👍